tutorials/spa-create-forms/spa-create-forms.md

```diff
@@ -233,44 +233,65 @@ These notifications will appear in the inbox of the requester as a task.
 
     > If copy is not available then create the form in the same way and modify the texts wherever relevant as shown below.
 
-8. Add the new form to the rejection route of the approval form.
+8. To add the new rejection form , we will use the **Duplicate** feature.
```

Comment:
Please don't use the "royal we" unless you are referring to SAP. When addressing the user, use "you" instead.